backend/apps/notification/services.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from .models import Notification
+
+
+
+def create_notification(user, notification_type, content, is_important=False):
```

Comment:
Reformat to follow the same convention as the ones in communities app. Ensure you use types, and instead of doing .save() and creating the model itself, just use <Model>.objects.create()